docs/src/examples/components/Animation/Usage/AnimationExampleVisible.shorthand.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react'
+import { Provider, Animation, Icon, Button } from '@stardust-ui/react'
+import { useLogKnob } from '@stardust-ui/docs-components'
+import { easeEasy } from 'src/themes/teams/animations/timingFunctions'
+
+const AnimationExampleVisible = () => {
+  const [visible, setVisible] = React.useState(false)
+
+  const onEnter = useLogKnob('onEnter', () => {})
```

Comment:
```suggestion
  const onEnter = useLogKnob('onEnter')
```

I missed that thing, let's update `useLogKnob()`:

```diff
  const proxy = React.useCallback<any>(
    (...a) => {
      appendLog(formatter(name, ...a))
-      return typeof callback === 'function' ? (callback as any)(...a) : null   
+      if (typeof callback === 'function') {
+        return (callback as any)(...a)
+      }
+
+      if (process.env.NODE_ENV !== 'production') {
+       console.log('Please provide a function to "useLogKnob(name, callback)"')
+      }
+      
+      return null
    },
    [appendLog, callback, name, formatter],
  )
```